api/src/test/java/org/apache/cloudstack/api/command/user/snapshot/CreateSnapshotPolicyCmdTest.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.cloudstack.api.command.user.snapshot;
```

Comment:
license needed, @shwstppr . Can you add it?

Reply:
@DaanHoogland added license